content/docs/integrations/github-org/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Configuring the GitHub Organization
```

Comment:
I think "Loading Github Organization Teams", like you have in the sidebar, is a better name. "Configuring the GitHub Organization" doesn't tell me why I should be interested in this tutorial.